modules/navier_stokes/test/tests/finite_volume/ins/action/restart-block-restriction/tests

```diff
@@ -9,16 +9,15 @@
                   "on a mesh which doesn't have a default block."
     recover = false
   []
+  # note that we are working to enable this, where the block restriction of the variable could be extended
+  # OR the action could live on a smaller domain than the variable
   [restricted-data-error]
     type = 'RunException'
     input = ns-restart-transient.i
     requirement = 'The system shall throw an error if the block-restrictions of the external variable and the action are different.'
-    cli_args = "Modules/NavierStokesFV/block='1 2' "
+    cli_args = "Modules/NavierStokesFV/block='1' Variables/pressure/block=1 "
                "Problem/allow_initial_conditions_with_restart=true"
-    expect_err = "(The suppled variable \(vel_x\) does not have the same block-restriction as the "
-                 "NSFVAction. The restriction of the variable is: \(ANY_BLOCK_ID\) while the "
-                 "restriction for the action is: \(1, 2\)|Physics 'NavierStokesFV' and object "
-                 "'vel_x' have different block restrictions.)"
+    expect_err = "Block restriction of interpolator user object 'ins_rhie_chow_interpolator' \(1\) doesn't match the block restriction of variable 'vel_x'"
```

Comment:
I guess this answers to my comment earlier in `PhysicsBase`; you're simply choosing not to support non-matching restrictions at this point, even if the variable is a superset

Reply:
Yes I m not sure this is needed at this point.